Dockerfile

```
@@ -2,6 +2,7 @@
 FROM golang:alpine as builder
 WORKDIR /app
 ADD . ./
+RUN apk add git
 RUN CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -installsuffix cgo -o aws-eks-asg-rolling-update-handler .
```

This is because the `alpine` tag points to 1.18 now, and IIRC a default behavior was removed which causes `go build` to fail unless you either add `--buildvcs=false` or add `git` to your docker image.

Since we have a `vendor` folder, could you replace `-mod vendor` by `--buildvcs=false` instead of installing `git`?
```suggestion
RUN CGO_ENABLED=0 GOOS=linux go build --buildvcs=false -a -installsuffix cgo -o aws-eks-asg-rolling-update-handler .
```